modules/developer_manual/pages/core/acceptance-tests.adoc

```diff
@@ -101,20 +101,22 @@ The testing app must be installed and enabled on the system-under-test.
 
 The testing app also provides skeleton folders that the tests can use as the default set of files for new users.
 
-[[apps/testing/data/apiSkeleton]]
+[[apps-testing-data-apiSkeleton]]
```

Comment:
the slash character is not "a good thing" here - it messes with the rendered document

Reply:
Custom anchor's aren't, strictly, required as anchors are auto-generated for headers.

Reply:
will it generate a valid anchor for a header that contains slashes?